### ISSUE TITLE
Storj Share GUI 7.1.0 in edge fails to launch

### ISSUE THREAD

**Issue by flexiondotorg** [snapcrafters/storjshare-gui#4]
When attempting to start Storj Share GUI from the edge channel is fails with:

    /snap/storjshare-gui/21/opt/storjshare/storjshare: error while loading shared libraries: libgtk-x11-2.0.so.0: cannot open shared object file: No such file or directory

**Comment by Ads20000** [comment]
Yeah this is a regression from 6.4.0 and works fine in the 7.1.0 Deb. Strange, because the `stage-package` is there :(

**Comment by popey** [comment]
I tried triggering a new build today to see if the updates to the desktop launchers would fix this, but it sadly doesn't. I think we're going to need to add a wrapper script which sets the LD_LIBRARY_PATH to coerce the binary to find the libs.

**Comment by popey** [comment]
I have closed the edge channel for now, as this snap is broken.
